src/cow-react/modules/limitOrders/pure/InfoBanner/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as styledEl from './styled'
+import AlertIcon from 'assets/cow-swap/alert-circle.svg'
+import SVG from 'react-inlinesvg'
+import { HashLink } from 'react-router-hash-link'
+import { useState, useEffect } from 'react'
+
+export function InfoBanner() {
+  // Explainer banner for orders
+  const LOCAL_STORAGE_KEY = 'limitOrders_showInfoBanner'
+  const [showBanner, setShowBanner] = useState(() => {
+    const item = localStorage.getItem(LOCAL_STORAGE_KEY)
+    return item !== null ? item === 'true' : true
+  })
+
+  const closeBanner = (): void => {
+    setShowBanner(false)
+    localStorage.setItem(LOCAL_STORAGE_KEY, 'false')
```

Comment:
Do we need this? Bellow we have useEffect that does the same

Reply:
Feel free to re-factor as I'm unsure what is needed or not.